contracts/mocks/modules/Assessment/ASMockAssessment.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity >=0.5.0;
+
+import {IAssessment} from "../../../interfaces/IAssessment.sol";
```

Comment:
to follow the convention, create a new folder under mocks/modules called "AssessmentViewer" and move this file there and call it AVMockAssessment - see CoverViewer for an example

Reply:
oh, i see! i completely misunderstood how the naming convention works. thanks